src/screen/edit_server.rs

```diff
@@ -142,14 +149,18 @@ impl super::Screen for EditServerEntry {
             let server_name = server_name.clone();
             let server_address = server_address.clone();
             done.add_click_func(move |_, game| {
-                Self::save_servers(
-                    index,
-                    &server_name.borrow().input,
-                    &server_address.borrow().input,
-                );
-                game.screen_sys
-                    .clone()
-                    .replace_screen(Box::new(super::ServerList::new(None)));
+                if &server_address.borrow().input == "" {
+                    save_server_error.borrow_mut().text = format!("Please enter a Server Address");
```

Comment:
Could you make this an early return instead of having an else branch?

Reply:
I have changed this to an early return. The `add_click_func()` requires the closure to return a bool, i cant find if the bool is used anywhere or if indicates anything, so i simply made it return false in case of error. From a quick test it seems to make no difference if True or False is returned.